src/drivers/px4io/px4io.cpp

```diff
@@ -1306,8 +1316,7 @@ int PX4IO::io_publish_raw_rc()
 		input_rc.input_source = input_rc_s::RC_INPUT_SOURCE_PX4IO_ST24;
 	}
 
-	if ((input_rc.channel_count > 0) && !input_rc.rc_lost && !input_rc.rc_failsafe
-	    && (input_rc.input_source != input_rc_s::RC_INPUT_SOURCE_UNKNOWN)) {
+	if (input_rc.input_source != input_rc_s::RC_INPUT_SOURCE_UNKNOWN) {
```

Comment:
Ok, so we no longer require rc_lost and rc_failsafe flags to be right to publish but we publish anyway and then check it further down the line, makes sense.